### DISCUSSION TITLE
Equipo 12 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#18] [Proyecto Final]
Hola @mitchida 
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar al Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @mitchida 

Gracias por entregar la etapa 0 del proyecto final. Estoy revisándola y te escribo por este medio con retroalimentación. 

Me parece muy importante el tema del proyecto, da un foco en la educación de una población con poca representación en muchas carreras. Las preguntas son bien construidas. Parece que entrevistarás a un docente en estas regiones, ¿es así? 

Algunas sugerencias: empieza con algo para romper el hielo (una pregunta, una historia, lo que sea). También importante avisar al entrevistado/a sobre la duración de la entrevista _antes_ de empezarla, para que esté preparado. Otra sugerencia sería preguntar sobre la experiencia de la persona (cuántos años ha enseñado en dicha región, ha enseñado a diferentes poblaciones o siempre a personas indígenas, etc). Al recolectar esta información, te apoyará a interpretar sus respuestas. Y finalmente, una curiosidad de mi parte: ¿sabes si los docentes que enseñan a poblaciones indigenas reciban entrenamiento? 

¿Has identificado una entrevistado/a? Mi recomendación sería agendar y completar la entrevista tan pronto como sea posible. Para que tengas tiempo suficiente para sintetizar tus apuntes y crear tu mapa (según los vídeos/la asesoría de esta semana). Solo tienes que crear una mapa, no es necesario crear todos los tipos que los vídeos explican.

¿Tienes preguntas para mí? Cualquier cosa, aquí ando.

¡Adelante!
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @mitchida 

Justo miré tu etapa 1 del proyecto final (aquí https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-12/etapa-1/README.md) y no veo la entrega ahí. Perdón si me equivoqué. ¿Cómo andas con tu entrevista y la creación de la mapa de síntesis? ¿Necesitas una prórroga? Te podría dar una prórroga hasta este domingo.

Quedo al pendiente,
Saludos,
Kasturi

**Reply by mitchida** [reply]
Hola @KasturiSShah,

He tenido problemas para realizar la entrevista. Quizá tenga que cambiar el tema, la persona a entrevistar  y las preguntas.
Espero poder hacer la entrega antes del domingo. Gracias

**Comment by mitchida** [comment]
Hola,

He tenido problemas para realizar la entrevista. Quizá tenga que cambiar el
tema, la persona a entrevistar  y las preguntas.
Espero poder hacer la entrega antes del domingo. Gracias

**Reply by KasturiSShah** [reply]
Hola @mitchida 
Gracias por la actualización. No te preocupes. Entendimos que agendar entrevistas puede ser difícil. De acuerdo que una opción podría ser cambiar el tema, la entrevista, y el entrevistado. Una otra opción sería elegir a alguien diferente para la entrevista (si tuvieras esta opción); de esa manera pedas usar el mismo tema, las mismas preguntas, etcetera.
Y claro, confirmo que tienes prórroga hasta el domingo. Si necesitas más tiempo, me avisas por aquí.
Mucha suerte
Kasturi

**Comment by KasturiSShah** [comment]
Hola @mitchida 
Espero que descansaste durante el feriado!
Gracias por entregar la etapa 1 del proyecto final. Veo el PDF y imagen en tu carpeta grupal.
Te escribo para avisar que la moderadora @Estefania-369 estará dándote retroalimentación para la etapa 1. Recibirás su retroalimentación por medio del hilo presente en el foro.
Una pregunta: para las etapas 2 y 3, ¿hasta que fecha quisieras una prórroga para las etapas 2 y 3? No es necesario entregar etapas pulidas, lo importante es practicar los conceptos enseñados en los módulos/las sesiones y aplicarlos a tu proyecto. 😄 
Saludos a la distancia,
Kasturi

**Comment by Estefania-369** [comment]
Hola @mitchida ! :+1:  

Gusto en saludarte, gracias por entregar la etapa 1 del proyecto final, así como tu dedicación en el desarrollo del mismo. Te escribo para compartir retroalimentación.

Es interesante el tema que estas desarrollando, a través de tu proyecto, al leer las preguntas, así como las respuestas que has sintetizado, me puedo dar cuenta que es un tema importante con oportunidades de mejora. 

Lo que se puede apreciar en la sinterización de las respuestas en los post-its utilizando el Mapa de afinidad, es que mantienes el hecho de la persona entrevistada, y no tu interpretación. Has agrupado bien las observaciones por tema, me parece que está bien la clasificación. Es entendible ya que también utilizaste diferentes colores para facilitar la identificación por temas. 
Se puede observar puntos clave que te servirán para generar lluvia de ideas y así identificar tu objetivo principal para dar solución a la educación en zonas rurales. 

Como sugerencia podrías agregar también un resumen de la entrevista, con puntos importantes para ti y que te llamaron la atención, estos los puedes tomar en cuenta cuando empieces a desarrollar la lluvia de ideas de la etapa 2, te podría facilitar para generar ideas, utilizando las estrategias que se ven en el modulo 2, como el 8s (ocho loco).

Te felicito por tu trabajo y dedicación. :D
Si tienes alguna duda, estamos para apoyarte.

Espero que estes teniendo una agradable semana, mucho éxito y mis mejores deseos.

Un abrazo
Estefania

**Comment by KasturiSShah** [comment]
Hola @mitchida 

Espero estés muy bien.

¡La clausura del curso es la semana que viene! Estamos en las últimas estrechas del curso. Sería un placer mirar tu presentación de este proyecto con el resto de las alumnxs ahí. Déjame saber si te conviene la asignación de tu equipo a una de las dos sesiones de clausura del curso, o si quieres cambiar su día asignado al otro día. 

**No es necesario tener un proyecto pulido ni completado para presentarlo. Sería excelente en la clausura del curso simplemente mostrar el trabajo que has hecho y describir los aspectos que habrías desarrollado más si hubieras tenido más tiempo.**

Estamos (@Estefania-369 y yo) aquí para apoyarte. Cualquier pregunta sobre tu proyecto, déjanos saber por medio de este hilo y te contestaríamos pronto.

Saludos,
Kasturi
